app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -7,11 +7,7 @@ package org.mozilla.focus.fragment
 import android.arch.lifecycle.Observer
 import android.os.Bundle
 import android.text.TextUtils
-import android.view.KeyEvent
-import android.view.LayoutInflater
-import android.view.View
-import android.view.ViewGroup
-import android.view.WindowManager
+import android.view.*
```

Comment:
This is weird – we should use not `*` imports. I wonder if the IDE just did this (because that's how it's configured by default) and checkstyle didn't complain because it wasn't configured to...

Anyway, I'm not that opinionated about not using * imports (I guess it's so you don't accidentally clobber the namespace?).

Reply:
yah the IDE just did this